schema.graphql

```diff
@@ -877,14 +934,26 @@ input UpdateEventInput {
   title: String
 }
 
+input UpdateEventProjectInput {
+  description: String
+  title: String
+}
+
 input UpdateOrganizationInput {
   description: String
+  isPublic: Boolean
   location: String
   name: String
-  userRegistrationRequired: Boolean
   visibleInSearch: Boolean
 }
 
+input UpdateTaskInput {
+  completed: Boolean
+  deadline: DateTime
```

Comment:
@duplixx 

1. We removed Projects and Tasks from the API a few PRs ago.  Your starting code is out of date
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1627
1. This is going to be replaced with Action Items.
    - https://github.com/PalisadoesFoundation/talawa-api/pull/1648
1. Remove this Type and all references to it